app/controllers/StoreMyNinoController.scala

```diff
@@ -48,7 +51,28 @@ class StoreMyNinoController @Inject()(
   def onPageLoad: Action[AnyContent] = (authorisedAsFMNUser andThen getPersonDetailsAction) async {
     implicit request => {
       auditService.audit(AuditUtils.buildAuditEvent(request.personDetails, "ViewNinoLanding", frontendAppConfig.appName, None))
-      Future(Ok(view(request.nino.map(_.formatted).getOrElse(""))))
+      val fullName: String = request.personDetails.person.fullName
+      val nino: String = request.nino.map(_.formatted).getOrElse("")
+      val googleId = googleWalletConnector.createGooglePass(fullName, nino)
```

Comment:
As these are futures not yet mapped I would name them 'googleIdf' and `appleIdf` respectively. This stops the names conflicting in the flat map and map.

Reply:
Oh okay no problem I can do that